src/sql/workbench/contrib/extensions/browser/extensionsActions.ts

```diff
@@ -85,3 +87,58 @@ export class OpenExtensionAuthoringDocsAction extends Action {
 		await this.openerService.open(URI.parse(OpenExtensionAuthoringDocsAction.extensionAuthoringDocsURI));
 	}
 }
+
+export class HidePanel extends Action {
+	static readonly ID = 'workbench.extensions.action.hidePanel';
+	static readonly LABEL = localize('hidePanel', "Hide the panel...");
+
+	constructor(
+		id: string = HidePanel.ID,
+		label: string = HidePanel.LABEL,
+		@IWorkbenchLayoutService private readonly layoutService: IWorkbenchLayoutService
+	) {
+		super(id, label);
+	}
+
+	override async run(): Promise<void> {
+		this.layoutService.setPanelHidden(true);
+	}
+}
+
+export class HideSettings extends Action {
+	static readonly ID = 'workbench.extensions.action.hideSettings';
+	static readonly LABEL = localize('hideSettings', "Hide the settings icon...");
+
+	constructor(
+		id: string = HideSettings.ID,
+		label: string = HideSettings.LABEL,
+	) {
+		super(id, label);
+	}
+
+	override async run(): Promise<void> {
+		let allActionItems = Array.from(document.getElementsByClassName('action-item icon'));
+		let manageElement = allActionItems.filter((el) => el.getAttribute('aria-label') === 'Manage');
+		manageElement[0].parentNode.removeChild(manageElement[0]);
```

Comment:
Instead of doing DOM manipulation why not just fine where the settings button is created and skip making it if the product quality is tsgops. That seems a lot simpler and less error prone.

(for example right now this assumes the aria-label is English which isn't necessarily true so it'll break on non-English installs)